debian/rules

```diff
@@ -84,10 +84,8 @@ endif
 	    $${MYSQL_COMPILER_LAUNCHER:+-DCMAKE_CXX_COMPILER_LAUNCHER=${MYSQL_COMPILER_LAUNCHER}} \
 	    $${MYSQL_COMPILER_LAUNCHER:+-DCMAKE_C_COMPILER_LAUNCHER=${MYSQL_COMPILER_LAUNCHER}} \
 	    -DCMAKE_SYSTEM_PROCESSOR=$(DEB_BUILD_ARCH) \
+	    -DCMAKE_CXX_LIBRARY_ARCHITECTURE=$(DEB_HOST_MULTIARCH) \
 	    -DBUILD_CONFIG=mysql_release \
-	    -DINSTALL_LIBDIR=lib/$(DEB_HOST_MULTIARCH) \
-	    -DINSTALL_PLUGINDIR=lib/mysql/plugin \
```

Comment:
Why remove INSTALL_PLUGINDIR here? Was it intentional?

Reply:
Because its exactly the same as INSTALL_PLUGINDIR_DEB in cmake/install_layout.cmake where all path overrides should occur . Same with INSTALL_MYSQLTESTDIR_DEB below.